charts/studio/templates/configmap-studio.yaml

```diff
@@ -109,11 +109,7 @@ data:
 
   SOCIAL_AUTH_REDIRECT_IS_HTTPS: "False"
 
-  {{- if .Values.global.ingress.enabled }}
   SOCIAL_AUTH_ALLOWED_REDIRECT_HOSTS: "studio-ui.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.studioUi.service.port }},studio-backend.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.studioBackend.service.port }},{{ .Values.global.host }}"
```

Comment:
`.Values.global.host` might be empty. I'm not sure that the backend can handle such a case correctly. Have you tested it?